apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.ts

```diff
@@ -6,16 +6,17 @@ import { MatSelectModule } from '@angular/material/select';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { TranslateService, TranslateModule } from '@ngx-translate/core';
 import { FormsModule } from '@angular/forms';
-import { Language } from '@iglidur-designer/interfaces';
+import { Languages, FlagUrls, LanguageCode } from '@iglidur-designer/interfaces';
 import { LanguageService } from '@iglidur-designer/services';
 import { LanguagesComponent } from '../languages/languages.component';
 import { Subject, Subscription, takeUntil } from 'rxjs';
 import { MatDividerModule } from '@angular/material/divider';
 import { MatRadioButton, MatRadioGroup } from '@angular/material/radio';
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
Why this?

Reply:
I was getting an error about the refactored code causing circular dependency issues but it wasn't so when I investigated it.
It started when I removed a block of code from one component into the utils library, so that I can reuse it in different components.

Reply:
If you are having a circular dependency error then it is a circular dependency error.  I don't think disabling is the right approach